### PR TITLE
Add a make command to generate release notes

### PR DESCRIPTION
**Pull request by tas50**
#### Description

I was wasting time copying the last one each time and then changing things to make it a template that I could use for the next week. Why not automate this a bit?

#### Types of changes

- [ ] Functional documentation bug fix (i.e., broken link or some other busted behavior)
- [x] New functional doc capabilities (i.e., filter search results)
- [ ] New content
- [ ] Revision to existing content
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

#### Checklist

- [x] I have read the **README** document about contributing to this repo.
- [x] I have tested my changes locally and there are no issues.
- [x] All commits are signed.

### CODE REVIEW

**Review by afiune — COMMENTED** [review]
Is this just a template? or, where are you generating the release notes? 🤔 

At my previous employer, I created this simple bash script that generates the release notes based of our **CLEAN** git history, not sure if we want to do something similar?

https://github.com/lacework/go-sdk/blob/main/scripts/release.sh#L254

File: Makefile [hunk 1]
```diff
@@ -18,6 +18,19 @@ build: yarn fmt
...
+###
+
+.PHONY: notes
+notes:
+	@echo "Generating new release note file\n\n"
+	@read -p 'What version should I use? ' VERSION; \
+	read -p 'What date will this release ship (ex: 2024-11-05)? ' DATE; \
+	cp ./releases/_release_template_file.md ./releases/$${DATE}-mondoo-$${VERSION}-is-out.md; \
+	mkdir -p ./static/img/releases/$${DATE}-mondoo-$${VERSION}-is-out; \
+	sed -i '' "s/VERSION/$${VERSION}/g" "./releases/$${DATE}-mondoo-$${VERSION}-is-out.md"
```
Comment (philipbalinov) [inline Makefile:32]: nit: what does that empty quoted string `''` passed to sed do?

File: Makefile [hunk 2]
```diff
@@ -18,6 +18,19 @@ build: yarn fmt
...
+###
+### Create a new release note
+###
+
+.PHONY: notes
+notes:
+	@echo "Generating new release note file\n\n"
+	@read -p 'What version should I use? ' VERSION; \
+	read -p 'What date will this release ship (ex: 2024-11-05)? ' DATE; \
+	cp ./releases/_release_template_file.md ./releases/$${DATE}-mondoo-$${VERSION}-is-out.md; \
```
Comment (philipbalinov) [inline Makefile:30]: shellcheck says - surround the second parameter with quotes, on the off chance that one of the variables contains whitespace

File: Makefile [hunk 3]
```diff
@@ -18,6 +18,19 @@ build: yarn fmt
...
+	mkdir -p ./static/img/releases/$${DATE}-mondoo-$${VERSION}-is-out; \
```
Comment (philipbalinov) [inline Makefile:31]: same as above

File: Makefile (hunk 1 above)
Comment (tas50) [reply Makefile:32]: That's a macOS style sed thing. The first arg is the backup path so folks leave it as an empty string.

**Review by philipbalinov — APPROVED** [review]

**Comment by tas50** [comment]
@afiune the release notes are marketing things mostly not a technical description of what ships so git commits don't really work. I always start by checking all the releases and obviously the better folks write their commits, the easier it is on me, but it would be pretty hard to automate at that level. It needs a human touch.

**Pull request merged**